test/js-api/tag/type.tentative.any.js

```diff
@@ -0,0 +1,21 @@
+// META: global=window,dedicatedworker,jsshell
+// META: script=/wasm/jsapi/assertions.js
+
+function assert_type(argument) {
+    const tag = new WebAssembly.Tag(argument);
```

Comment:
It looks some functions use two spaces while others use four spaces. What's our rule for other JS spec tests? It'd better be consistent.

Reply:
Thanks for noticing that, I think 2-space is the norm based on other tests. I've just reformatted all the files I added with prettier to be consistent. @Ms2ger is that fine for spec test style?